src/plugins/controls/public/range_slider/components/range_slider_popover.tsx

```diff
@@ -143,8 +146,8 @@ export const RangeSliderPopover: FC = () => {
         <EuiFlexItem>
           <EuiDualRange
             id={id}
-            min={hasAvailableRange ? rangeSliderMin : undefined}
-            max={hasAvailableRange ? rangeSliderMax : undefined}
+            min={hasAvailableRange ? rangeSliderMin : 0}
+            max={hasAvailableRange ? rangeSliderMax : 100}
```

Comment:
Tested this by getting the range slider control into a no-data state, and it looks the same as it did before. 👌